opencti-platform/opencti-front/src/private/components/entities/individuals/IndividualKnowledge.js

```diff
@@ -77,6 +77,28 @@ class IndividualKnowledgeComponent extends Component {
             ))
             }
           />
+          <Route
+            exact
+            path="/dashboard/entities/individuals/:individualId/knowledge/targets"
+            render={(routeProps) => (
+              <EntityStixCoreRelationships
+                entityId={individual.id}
+                relationshipTypes={['targets']}
+                isRelationReversed
+                entityLink={link}
+                allDirections={true}
+                stixCoreObjectTypes={[
```

Comment:
According to the stix.ts file, it's missing the type Attack-Pattern and the campaign type is not available.

Reply:
Maybe it's better to don't pass any entities ?